src/commands/disclaimer.rs

```diff
@@ -1,23 +1,29 @@
+use db::sea_orm::{EntityTrait, Set, ActiveModelTrait};
 use serenity::{
     framework::standard::{macros::command, Args, CommandResult},
     model::channel::Message,
     prelude::Context,
     futures::StreamExt, utils::Colour
 };
 
-use crate::db::{get_guild, update_compliancy};
+use crate::Connection;
 
 const DISSEMINATION: &str = "The use of the snapshot command requires logging of information which must be disseminated to clients of this bot and of which they must agree to. Chain will not locally but, instead, within the guild, log this information from the member: usernames, discriminators, client IDs and messages sent within this server. It will send this data in bytes that will form a file at some designated channel; this file is never stored anywhere but within the server. It will not do anything more with the information gathered and to repeat, it will not store it locally.";
 
 #[command] 
 #[num_args(0)]
 #[aliases(dissemination)]
 async fn disclaimer(ctx: &Context, msg: &Message, _: Args) -> CommandResult{
-    let _guild_info = match get_guild(msg.guild_id.unwrap().as_u64()) 
+
+    let guild_id = *msg.guild_id.unwrap().as_u64();
+    let data = ctx.data.read().await;
+    let con = data.get::<Connection>().unwrap();
+
+    let mut guild: db::guild::ActiveModel = match db::guild::Entity::find_by_id(guild_id as i64).one(con).await
```

Comment:
is there a reason we can't use this syntax?
let pear: Option<fruit::Model> = Fruit::find_by_id(1).one(db).await?;

Reply:
they are exactly the same thing for the most part however `Option<db::guild::Model>` would not work as we do need to edit the guild which `ActiveModel` does allow

Reply:
i was just saying long term, since we're setting patterns, it might be less annoying to type `Guild::find_by_id` rather than `db::guild::Entity::find_by_id`

but i'll defer to you

Reply:
actually I've just realised they are the same thing completely. to fix this instead we should just import `db::guild::Entity` so that we can drop the `db::guild`

(the `Fruit::find_by_id` is referencing essentially `entities::fruit::Entity` or whatever the equivalent is)